util/utils.go

```diff
@@ -28,7 +28,8 @@ var (
 		return os.Remove(path)
 	}
 
-	InsideGUI = func() bool { return false }
+	InsideGUI    = func() bool { return false }
+	ShellTimeOut = 3 * time.Minute
```

Comment:
Can we set this to ~5 minutes? The ideal solution would be to bail if we aren't making progress instead of setting a hard cutoff, but that's going to be a bit harder to do (although you're welcome to try to implement that).

Unfortunately, a 3m cutoff will make it impossible to update IPFS on a 1Mbps connection.

Reply:
@Stebalien yes,i changed the commit.